docs/contents/components/forms/form-control/accessibility.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: FormControl
+description: "`FormControl` is a component used to group form elements with label, helper message, error message, etc."
+package_name: "@yamada-ui/form-control"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+## Keyboard Interaction
+
+| Key   | Description          | State |
+| ----- | -------------------- | ----- |
+| `Tab` | Focuses the element. | -     |
+
+## ARIA Roles and Attributes
+
+| Component           | Roles and Attributes  | Usage                                                          |
+| ------------------- | --------------------- | -------------------------------------------------------------- |
+| `FormControl`       | `aria-disabled`       | Set to true when `isDisabled` is specified.                    |
+|                     | `aria-invalid`        | Set to true when `isInvalid` is specified.                     |
+|                     | `aria-readonly`       | Set to true when `isReadOnly` is specified.                    |
+|                     | `aria-required`       | Set to true when `isRequired` is specified.                    |
+| `RequiredIndicator` | `role="presentation"` | Indicates that this is a presetntation.                        |
```

Comment:
In `RequiredIndicator`, there are cases where `aria-hidden` is set, so this should be explained.

Reply:
Fixed. Thanks for catching this! :)